helm-charts/mutator/values.yaml

```diff
@@ -38,6 +38,9 @@ namespaceService: default # namespace of cluster
 port: 443 # port of this service
 targetPort: 8443 # port to accept traffic on
 
+injectionNamespaces:
+  - default
```

Comment:
maybe it will be better to use deployment namespace as default namespace?
e.g. `{{ .Release.Namespace }}`